GUI/Main.Designer.cs

```diff
@@ -581,27 +581,22 @@ private void InitializeComponent()
             this.ModInfoTabControl.Name = "ModInfoTabControl";
             this.ModInfoTabControl.Size = new System.Drawing.Size(360, 836);
             this.ModInfoTabControl.TabIndex = 0;
-            // 
-            // StatusPanel
-            // 
-            this.StatusPanel.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
-            this.StatusPanel.Controls.Add(this.StatusLabel);
-            this.StatusPanel.Location = new System.Drawing.Point(0, 1074);
-            this.StatusPanel.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
-            this.StatusPanel.Name = "StatusPanel";
-            this.StatusPanel.Size = new System.Drawing.Size(1196, 29);
-            this.StatusPanel.TabIndex = 8;
-            // 
+            //
             // StatusLabel
-            // 
-            this.StatusLabel.Dock = System.Windows.Forms.DockStyle.Fill;
-            this.StatusLabel.Location = new System.Drawing.Point(0, 0);
-            this.StatusLabel.Margin = new System.Windows.Forms.Padding(4, 0, 4, 0);
+            //
             this.StatusLabel.Name = "StatusLabel";
-            this.StatusLabel.Size = new System.Drawing.Size(1050, 29);
-            this.StatusLabel.TabIndex = 0;
+            this.StatusLabel.Spring = true;
+            this.StatusLabel.Text = "";
             this.StatusLabel.TextAlign = System.Drawing.ContentAlignment.MiddleLeft;
-            // 
+            //
+            // StatusProgress
+            //
+            this.StatusProgress.Enabled = true;
+            this.StatusProgress.Visible = false;
+            this.StatusProgress.Minimum = 0;
+            this.StatusProgress.Maximum = 100;
+            this.StatusProgress.Size = new System.Drawing.Size(300, 20);
+            this.StatusProgress.Alignment = System.Windows.Forms.ToolStripItemAlignment.Right;
```

Comment:
Add `this.StatusProgress.Name = "StatusProgress";`

Reply:
Done